datastacks/datastacks_cli.py

```diff
@@ -0,0 +1,38 @@
+import logging
+import shutil
+import yaml
+
+import click
+from click_loglevel import LogLevel
+
+from datastacks.logger import setup_logger
+from datastacks.utils import render_template_component
+
+
+@click.group()
+@click.option("--log-level", "-l", type=LogLevel(), default=logging.INFO)
+def cli(log_level):
+    setup_logger("datastacks", log_level)
+
+
+@click.command()
+@click.option("--config-path", "-c", type=str, help="Absolute path to config file on local machine")
+@click.option("--data-quality/--no-data-quality", "-dq/", default=False, help="Flag to determine whether to include data quality in template")
```

Comment:
Just to confirm I understand this:
the options `--data-quality` or `-dq` = DQ included
the option `--no-data-quality` or leaving out the option altogether = DQ not included

Reply:
Yup spot on, I can add a shorthand for no data quality aswell if we want though, maybe -ndq?

Reply:
Sounds good, just so it's explicit

Reply:
Done